HelpSource/Classes/ScopeView.schelp

```diff
@@ -20,11 +20,25 @@ INSTANCEMETHODS::
 METHOD:: bufnum
     The number of the Buffer to plot.
 
-    As soon as a valid buffer number is set and a link::Classes/ScopeOut:: UGen is writing into it, the view starts periodically plotting the buffer. If the ScopeOut UGen stops writing, or an invalid buffer number is set, the plotting will pause.
+    To set up the ScopeView object for plotting, one needs to set a valid buffer number and tell a link::Classes/ScopeOut2:: UGen to write to it. Before the view starts periodically plotting the buffer, however, one needs to assign it a link::Classes/Server:: object and call the link::#-start:: method on it.
+
+If the ScopeOut2 UGen stops writing or an invalid buffer number is set, the plotting will pause.
```

Comment:
Indent this line to match the ones above it.

Reply:
Thanks for you patience with this! There's still a small indentation issue to take care of and then you should be done.